src/app/content/highlights/components/SummaryPopup/HighlightListElement.tsx

```diff
@@ -141,11 +142,13 @@ const HighlightListElement = ({ highlight, locationFilterId, pageId }: Highlight
       onColorChange={updateColor}
     />}
     <HighlightContentWrapper color={highlight.color}>
-      <ContentExcerpt
-        data-highlight-id={highlight.id}
-        content={highlight.highlightedContent}
-        source={highlight.sourceId}
-      />
+      <DynamicContentStyles>
```

Comment:
this is still rendering an extra div here. you may want to change to something more extensible like a higher order component or function-as-child pattern that would allow you to just compose the classname onto other elements

Reply:
i haven't looked in contentExcerpt but maybe you could just update that too